### DISCUSSION TITLE
sum by hrs

### DISCUSSION THREAD

**Discussion by Viv1892** [dbeaver/dbeaver#21736] [DBeaver Development]
Hi,

I have a query which gives me total qty scanned and timestamp. I would like to get total qty for each hour. I’m writing this in DBeaver which is my first time using this platform. Can anyone please help me how to sum qty hourly? I have attached the img for reference. This data goes for more than 3 months.

Thanks!
[sum by hours.xlsx](https://github.com/dbeaver/dbeaver/files/13242885/sum.by.hours.xlsx)

![image](https://github.com/dbeaver/dbeaver/assets/149711415/b7da23b9-49a9-423e-a8c8-f18b163db0a5)

**Comment by LonwoLonwo** [comment]
Hello @Viv1892 

Maybe some of the Panels will be helpful in your case: https://github.com/dbeaver/dbeaver/wiki/Panels
Grouping or Calc.
